includes/admin/class-wp-job-manager-cpt.php

```diff
@@ -334,11 +334,8 @@ public function jobs_by_category() {
 				'class'    => [],
 			],
 		];
-
-		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No changes or data exposed based on input.
-		$selected_category = isset( $_GET['job_listing_category'] ) ? sanitize_text_field( wp_unslash( $_GET['job_listing_category'] ) ) : '';
-		echo "<select name='job_listing_category' id='dropdown_job_listing_category'>";
-		echo '<option value="" ' . selected( $selected_category, '', false ) . '>' . esc_html__( 'Select category', 'wp-job-manager' ) . '</option>';
+		echo "<select name='" . esc_attr( \WP_Job_Manager_Post_Types::TAX_LISTING_CATEGORY ) . "' id='dropdown_job_listing_category'>";
```

Comment:
Do we also want to use the constant in a place like this?  `id='dropdown_job_listing_category'`?

Reply:
I thought about doing so, but I think the main issue with that is that `id` is the sort of attribute that might be used from the CSS/JS side to handle the field somehow (e.g., select2 stuff, for instance). 

And given that passing these constants to the JS side just to handle this looks a bit..too much, I decided just to leave it as it is. Right now, this whole "replacing magical strings with constants" looks already challenging enough (in the sense of risk of breaking things)